src/modules/sensors/sensor_params.c

```diff
@@ -43,10 +43,13 @@
  * Tube Pressure Drop
  * 		CAL_AIR_TUBED_MM: Diameter in mm of the pitot and tubes, must have the same diameter.
  * 		CAL_AIR_TUBELEN: Length of the tubes connecting the pitot to the sensor and the static + dynamic port length of the pitot.
+ * Venturi effect sensor
+ *    Supports TFSLOT airspeed sensor
  *
  * @value 0 Model with Pitot
  * @value 1 Model without Pitot (1.5 mm tubes)
  * @value 2 Tube Pressure Drop
+ * @value 3 Venturi effect airspeed sensor
```

Comment:
```suggestion
 * @value 3 Venturi effect airspeed sensor
```

Should this be more specific?

Reply:
I slightly expanded the description of the parameter. What exactly would you imagine here?